env.sh

```diff
@@ -20,4 +20,5 @@ LOG_FILE="/var/log/azureDiskInspectSvc.log"
 CREDSCANSERVICEVOLUMENAME="CredScanSvc"
 
 # TODO: replace this with envionment specific AppInsights instrumentation key
-APPINSIGHTS_KEY='00000000-0000-0000-00000000000000000'
\ No newline at end of file
+APPINSIGHTS_KEY='00000000-0000-0000-00000000000000000'
+APPINSIGHTS_ENDPOINTURL='https://dc.services.visualstudio.com/v2/track'
```

Comment:
What is this endpoint url for? Could you add a comment?

Reply:
Is this the default value for Prod? Let's have a comment on its purpose and if this is the fallback value for a parameterized input.

Reply:
And where you expect the parameterized input to come from.

Reply:
Added comment.
This comes from deployment scripts.